src/style/panel.less

```diff
@@ -133,12 +133,11 @@
 }
 
 .formline__no-padding {
-    padding-top: 0.2rem;
     padding-bottom: 0.1rem;    
 }
 
 .formline:first-child {
-    padding-top: 0;
+    padding-top: 0rem;
```

Comment:
I don't think this is valid CSS? It probably works either way, but I was under the impression that 0 should always be unit-less.